MAUI/DataForm/CustomDataFormItem.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Custom DataFormItem in Maui DataForm control | Syncfusion
```

Comment:
Manually create DataFormItems in .Net MAUI DataForm control | Syncfusion

Reply:
title changed